99-test-file

```diff
@@ -0,0 +1,6 @@
+# Test page
```

Comment:
That sounds like a great idea
I've been experimenting with animated SVGs lately which might be useful for this.
Let's chat soon!